enterprise/frontend/src/embedding-sdk/config.ts

```diff
@@ -1,5 +1,7 @@
 export const DEFAULT_FONT = "Lato";
 export const EMBEDDING_SDK_ROOT_ELEMENT_ID = "metabase-sdk-root";
+export const EMBEDDING_SDK_PORTAL_CONTAINER_ELEMENT_ID =
+  "metabase-sdk-portal-container";
```

Comment:
I think you should still name this portal root or something. Since you're only rendering this portal container once per Metabase provider.

I think container has a broader meaning and there could be more than 1 containers like one container per popover for example, but root feels like there should be only a single root.

Another reason is that it makes this easier to understand even if a bit if you have both metabase root and portal root, you know they're roots for different purposes.

tl;dr it's just a naming thing, but I think `root` fits better.

Reply:
I'm fine with either name, but lest just remember that since we're using ids, I'm not exactly sure what will happen if have two providers on the same page.
It's probably ok if the theme passed to both is the same

Reply:
> but lest just remember that since we're using ids, I'm not exactly sure what will happen if have two providers on the same page.

When that happens, slapping some number prefix should help 😄 I'm not too concerned though.